Makefile

```diff
@@ -22,6 +22,8 @@ setup-self:
 	python setup.py sdist && pip install dist/xblock-dalite-0.1.tar.gz
 
 test:
+	mkdir -p var
+	rm -r .coverage
```

Comment:
@jbzdak Confirmed.  I ran the tests, commented out all the tests, and then re-ran, and the coverage still said 100% as you stated.  Thanks for finding that!